torchtitan/parallelisms/parallelize_llama.py

```diff
@@ -60,7 +61,7 @@ def _custom_policy(mode, func, *args, **kwargs):
 
         def selective_checkpointing_context_fn():
             meta = defaultdict(int)
-            return _pt2_selective_checkpoint_context_fn_gen(_get_custom_policy(meta))
```

Comment:
Hey @wanchaol, I think this PR (or https://github.com/pytorch/torchtitan/pull/397) might have broken SAC + compile. I didn't verify though since `_pt2_selective_checkpoint_context_fn_gen` isn't in PyTorch trunk anymore. Do you think it's due to this change, or SAC + compile was already broken?